scripts/scenarios/cloud6/cloud7-upgrade-non-disruptive-ceph.yml

```diff
@@ -93,6 +93,7 @@ proposals:
         admin_keyring: "/etc/ceph/ceph.client.admin.keyring"
         pool: volumes
         user: cinder
+        secret_uuid: 75ffb452-2478-4c4c-abcc-1be39cb6325a
```

Comment:
1. That's what I havee  in one of my setup.
2. No idea
3. I think it should, yes